docs: box rw access, mutable app setting, crypto opcodes, block load and congestion, lsig and txn pay-per-size, hb discount (#294)
algorandfoundation/specs: src/ledger/ledger-parameters.md

## Block

|                 Parameter                 |     Current Value     |  Unit   | Description                                                           | Reference Implementation Name      |
|:-----------------------------------------:|:---------------------:|:-------:|-----------------------------------------------------------------------|------------------------------------|
|       \\( \MaxTxnBytesPerBlock \\)        | \\( 5{,}242{,}880 \\) |  bytes  | Maximum number of transaction bytes in a block                        | `MaxTxnBytesPerBlock`              |
|      \\( \MaxTimestampIncrement \\)       |      \\( 25 \\)       | seconds | Maximum difference between successive timestamps                      | `MaxTimestampIncrement`            |
|       \\( \MaxVersionStringLen \\)        |      \\( 128 \\)      |  bytes  | Maximum length of protocol version strings                            | `MaxVersionStringLen`              |
| \\( \MaxProposedExpiredOnlineAccounts \\) |      \\( 32 \\)       |         | Maximum number of _expired participation accounts_ in block header    | `MaxProposedExpiredOnlineAccounts` |

> |       \\( \PayoutMaxMarkAbsent \\)        |      \\( 32 \\)       |         | Maximum number of _suspended participation accounts_ in block header  | `Payout.MaxMarkAbsent`             |

Reviewer: it is `Payouts.MaxMarkAbsent`, an 's' is missing, worth fixing `PayoutMaxMarkAbsent` as well

|         \\( \RewardUnit \\)          |       \\( 1{,}000{,}000 \\)        | μALGO  | Size of an earning unit                                                                                                             | `RewardUnit`                   |
| \\( \RewardsRateRefreshInterval \\)  |         \\( 500{,}000 \\)          | rounds | Rate at which the reward rate is refreshed                                                                                          | `RewardsRateRefreshInterval`   |

## Protocol Upgrade

|             Parameter             |   Current Value   |  Unit  | Description                                               | Reference Implementation Name |
|:---------------------------------:|:-----------------:|:------:|-----------------------------------------------------------|-------------------------------|
| \\( \DefaultUpgradeWaitRounds \\) | \\( 140{,}000 \\) | rounds | Default number of rounds needed to prepare for an upgrade | `DefaultUpgradeWaitRounds`    |
|   \\( \MaxUpgradeWaitRounds \\)   | \\( 250{,}000 \\) | rounds | Maximum number of rounds needed to prepare for an upgrade | `MaxUpgradeWaitRounds`        |

> |  \\( \\MinUpgradeWaitRounds \\)   | \\( 10{,}000 \\)  | rounds | Minimum number of rounds needed to prepare for an upgrade | `MinUpgradeWaitRounds`        |

Reviewer 2: ```suggestion
|  \\( \MinUpgradeWaitRounds \\)   | \\( 10{,}000 \\)  | rounds | Minimum number of rounds needed to prepare for an upgrade | `MinUpgradeWaitRounds`        |
```

| \\( \MaxAssetUnitNameBytes \\) |   \\( 8 \\)   | bytes | Maximum length of asset unit (symbol)         | `MaxAssetUnitNameBytes`       |
|   \\( \MaxAssetURLBytes \\)    |  \\( 96 \\)   | bytes | Maximum length of asset URL                   | `MaxAssetURLBytes`            |

## LogicSig

|                Parameter                |  Current Value   |  Unit   | Description                                                                                                                                                     | Reference Implementation Name    |
|:---------------------------------------:|:----------------:|:-------:|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------|
|        \\( \LogicSigMaxCost \\)         | \\( 20{,}000 \\) | opcodes | Maximum opcode cost for LSig                                                                                                                                    | `LogicSigMaxCost`                |

> |        \\( \LogicSigMaxSize \\)         | \\( 1{,}000 \\)  |  bytes  | Maximum combined length of LSig program and LSig arguments                                                                                                      | `LogicSigMaxSize`                |

Reviewer: ```suggestion
|        \\( \LogicSigMaxSize \\)         | \\( 1{,}000 \\)  |  bytes  | Unit used to compute two independent group-wide allowances: the pooled LSig argument allowance and the free program-byte allowance                              | `LogicSigMaxSize`

|    \\( \MaxLocalSchemaEntries \\)     |    \\( 16 \\)     |              | Maximum number of key/value pairs of application local state                                                                      | `MaxLocalSchemaEntries`        |
|     \\( \MaxInnerTransactions \\)     |    \\( 16 \\)     |     txn      | Maximum number of inner transactions for an `appl` transaction                                                                    | `MaxInnerTransactions`         |
|    \\( \SchemaBytesMinBalance \\)     | \\( 25{,}000 \\)  |    μALGO     | Additional MBR for `[]bytes` values in application state                                                                          | `SchemaBytesMinBalance`        |
|   \\( \SchemaMinBalancePerEntry \\)   | \\( 25{,}000 \\)  |    μALGO     | MBR for key-value pair in application state                                                                                       | `SchemaMinBalancePerEntry`     |

> |     \\( \SchemaUintMinBalance \\)     |  \\( 35{,}00 \\)  |    μALGO     | Additional MBR for `uint64` values in application state                                                                           | `SchemaUintMinBalance`         |

Reviewer 2: ```suggestion
|     \\( \SchemaUintMinBalance \\)     |  \\( 3{,}500 \\)  |    μALGO     | Additional MBR for `uint64` values in application state                                                                           | `SchemaUintMinBalance`         |
```
